packages/block-library/src/query/edit/inspector-controls/enhanced-pagination-control.js

```diff
@@ -15,7 +15,7 @@ export default function EnhancedPaginationControl( {
 	clientId,
 } ) {
 	const enhancedPaginationNotice = __(
-		'Enhanced pagination requires all descendants to be Core blocks. If you want to enable it, you have to remove all third-party blocks contained inside the Query Loop block.'
+		'Enhanced pagination only works with the blocks bundled with WordPress. If you want to enable it, you have to remove all plugin blocks from the Query Loop.'
```

Comment:
Something similar here:

```suggestion
		'Enhanced pagination doesn\'t support plugin blocks yet. If you want to enable it, you have to remove all plugin blocks from the Query Loop.'
```

cc: @jasmussen

Reply:
Nice.